agenta-backend/agenta_backend/models/api/evaluation_model.py

```diff
@@ -21,6 +21,7 @@ class Evaluation(BaseModel):
     status: str
     evaluation_type: EvaluationType
     evaluation_type_settings: Optional[EvaluationTypeSettings]
+    llm_app_prompt_template: Optional[str]
```

Comment:
What does this parameter do? how different it is from the evaluation_prompt_template?